cdc_stream/main.cc

```diff
@@ -18,7 +18,21 @@
 #include "cdc_stream/stop_service_command.h"
 #include "lyra/lyra.hpp"
 
-int main(int argc, char* argv[]) {
+int wmain(int argc, wchar_t* argv[]) {
```

Comment:
This makes the code less portable, not more portable. May I suggest to make this conditional on `#ifdef PLATFORM_WINDOWS`? We used to have it this way in our old code base, take a look at `manifest_creator/main.cc`.

Reply:
Done.